CMakeLists.txt

```diff
@@ -1,5 +1,16 @@
 cmake_minimum_required( VERSION 2.8.1 )
 
+# ----------------------------------------
```

Comment:
See [forum discussion](https://icl.cs.utk.edu/magma/forum/viewtopic.php?f=2&t=3890&sid=ec9f1c094bae2c7e4083654a66106c24) for the original advice